app/controllers/api/users_controller.rb

```diff
@@ -2,6 +2,19 @@
 class Api::UsersController < Api::BaseController
   before_action :authorize_resource!
 
+  def show
+    render json: User.find(params.require(:id))
+  end
+
+  def show_via_resource
```

Comment:
use `/users?email=foo` ... don't need a copy-pasted endpoint ... also trying to get rid of /api